src/Feature/FileDrop/Feature_FileDrop.re

```diff
@@ -0,0 +1,3 @@
+[@deriving show({with_path: false})]
```

Comment:
This might seem small for now, but this is the perfect place for us to add a model once we incorporate dragging 👍

Reply:
I have to admit I'm a bit more concerned. Not so much because of the granularity, although that is a minor concern too, but more because this seems to be a cross-cutting concern. It's not clear to me how this will integrate with other features that need to support drag and drop. Why would dropping a file on the file explorer, for example, even need to involve a separate file drop feature project? It seems like very little code would live here, but would instead have to be implemented in the consuming feature projects (file explorer/workspace) or in the glue code, as is largely the case here.

I can see that there's a missing feature project here for this specific feature to live in however, but I think that missing project is for the workspace as a whole.

Reply:
> Not so much because of the granularity, although that is a minor concern too, but more because this seems to be a cross-cutting concern. It's not clear to me how this will integrate with other features that need to support drag and drop. Why would dropping a file on the file explorer, for example, even need to involve a separate file drop feature project?

Right, I think this is a fair concern! It seems like - at least the way this is implemented right now - `FilesDroppedOnEditor` - makes sense to be part of the `Feature_Editor` (since it's handling the use-case where a file is dropped on the editors, and not on the explorer or other parts of the shell, at the moment).

The main issue is that `Editor` is one of our least-factored features at the moment - we don't have a central `msg` or `updater` yet.

We could start a `Feature_Editor.msg` with `FilesDropped`, and stub out an updater - and then incrementally move the remaining functionality towards that (ie, there are lots of `Editor*` actions in `Actions.t` that could be migrated to this `msg` later).